apps/client/src/pages/Dashboard/Groups/group/exploreGroupView/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { Text } from 'components'
+import styles from './exploreGroup.module.scss'
+import frame from './images/Frame.svg'
+import rightless from './images/rightless.svg'
+import TableComponent from '../table/TableComponent'
+// import TableComponent from '../table/TableComponent'
+
+const cardData = [
```

Comment:
Good job Rukky but pls note data wont come like this from the backend. Pls sync with backend guys.